MunchkinLevelCounter/UI/Players Editor/PlayersEditorViewController.swift

```diff
@@ -190,18 +118,144 @@ class PlayersEditorViewController: BaseViewController, UITableViewDelegate, UITa
         warningAlert.addAction(cancelAction)
         present(warningAlert, animated: true, completion: nil)
     }
-
-    func onPlayerStatus(position: Int, playing: Bool) {
-        let playerId = players[position].playerId
-        players[position].isPlaying = playing
-        presenter?.markPlayerAsPlaying(withObjectId: playerId, isPlaying: playing)
+    
+    fileprivate func showPlayerDeleteAlert(indexPath: IndexPath) {
+        let deleteAlert = UIAlertController(title: "Delete Player",
```

Comment:
@vshkl That's the reason why I wrote in the chat that some of my comments can be not relevant due to the state of the code.